@types/modules/pm/categories.d.ts

```diff
@@ -0,0 +1,24 @@
+declare interface PMCategoryUpdateRequest {
+    prizeCategoryName?: string;
+    prizeCategoryDescription?: string;
+    eligibility?: string;
+    companyId?: string;
+    approvalStatus?: "approved" | "rejected" | "inProgress" | "awaitingApproval" | "public";
+  }
+  
+  declare interface PMCategoryCreateRequest extends PMCategoryUpdateRequest {
+    categoryId: string;
+    prizeCategoryName: string;
+    prizeCategoryDescription: string;
+  }
+  
+  declare interface PMCategory extends PMCategoryCreateRequest {
+    eligibility: string;
+    companyId: string;
+    approvalStatus: "approved" | "rejected" | "inProgress" | "awaitingApproval" | "public";

```

Comment:
If we keep this with "public", we'll have a 2-stage approval process; if we want to streamline it, it might be worth it to just display publicly any "approved" prizes.
```suggestion
    approvalStatus: "approved" | "rejected" | "inProgress" | "awaitingApproval";
```